apps/prairielearn/src/lib/editors.ts

```diff
@@ -921,12 +922,15 @@ export class QuestionAddEditor extends Editor {
   private qid?: string;
   private title?: string;
   private files?: Record<string, string>;
+  private isDraft?: boolean;
+  draftNumber: number;
```

Comment:
Why is this declared as a class member?

Reply:
removed in d830080b8b22c8440c018379442f76ca019db963.